Editor/Scripts/Tools/SelectionGroupTools.cs

```diff
@@ -24,9 +24,9 @@ static void ToggleVisibility(SelectionGroup group)
             SceneVisibilityManager sceneVisibilityManager = SceneVisibilityManager.instance;
             bool show = (sceneVisibilityManager.IsHidden(members[0]));
             if (show) {
-                sceneVisibilityManager.Show(members, false);
+                sceneVisibilityManager.Show(members, true);
```

Comment:
Are you sure that we want to toggle the visibility of the children as well ?
I can imagine that a person may want to control the visibility of objects in a SelectionGroup that contain a parent object and its children object.